torch/_dynamo/config.py

```diff
@@ -374,6 +374,12 @@ def _get_optimize_ddp_mode():
 # Inline inbuilt nn modules
 inline_inbuilt_nn_modules = not is_fbcode()
 
+# Re-enable old torch.compile/torch._dynamo.disable behavior,
+# where compile and disable have the same priority.
+# Killswitch in case internal breaks.
+# See https://github.com/pytorch/pytorch/pull/132926.
+old_compile_disable_behavior = False
```

Comment:
This isn't a useful killswitch. You haven't hooked up to a JK (which we can use to change this value without a code change), and your value needs to default to True (because that's how JKs work, they default to True)